src/store/metadata/stub/content/datasets.json

```diff
@@ -11,11 +11,11 @@
       "@type": "dcat:DatasetSeries",
       "identifier": "cpih",
       "title": "Consumer Price Inflation including owner occupiers' housing costs (CPIH)",
-      "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH) is a measure of inflation that includes costs associated with...",
-      "description": "The Consumer Prices Index including owner occupiers' housing costs (CPIH) is a measure of inflation that includes costs associated with...",
-      "issued": "2017-01-01T00:00:00Z",
-      "modified": "2017-01-01T00:00:00Z",
-      "next_release": "2019-04-19T09:30:00Z",
+      "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH).",
```

Comment:
Changes to summary and description are great, but they should still differ from the title to some extent.